server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/VacationMailet.java

```diff
@@ -0,0 +1,116 @@
+/****************************************************************
```

Comment:
The ordering problem comes from github that order commits from dates and not from commit history : 

```
c8d89cbf6c611572b4623db660ce7f1a98d9cccb JAMES-1717 Vacation should allow user to specify a subject
9ea5de55ac062e906ed2a504858b7cefd87e04e2 JAMES-1717 SMTP Integration tests for JMAP vacation feature
721e57cd9b743c1da027e23c12b2906f1589575f JAMES-1717 Vacation mailet should not respond to automatically sent messages
6a496622a2fd356fdb4e01136e1976e302916d5a JAMES-1717 Mailet sub-project should allow automatically generated mail detection
a7225d4729e712ecc488aa604b5031048ebe77c9 JAMES-1717 Ensure Vacation mailet in transport processor
72f3ddbd3ec0cd561a7a429f03a4584a1b38d888 JAMES-1717 Implement a JMAP vacation mailet
2d32ea6d3765b6fd3b6a3c6969be9a064a67e5d3 JAMES-1717 Vacation should indicate if it is active
3dbd7a206c8b6e6976e0fdc27e24b1dffeecba99 JAMES-1717 Improve MPT SMTP tests : an empty line needs to be added between headers and body
d81f169f61195a9eae25c6ed8738ab619e604286 JAMES-1717 Add missing licenses
```